### PR TITLE
[#IR-326] As a user, I can see Actual Hours of the sprint cards

### PR DESCRIPTION
**Pull request by Ravi2714**

### CODE REVIEW

File: apps/retrospective/services/sprint.go [hunk 1]
```diff
@@ -339,7 +340,7 @@ func (service SprintService) getSprintTaskTypeSummary(
 		QueryExpr()
 
 	taskQuery := taskList.Select(
-		"COUNT(DISTINCT(tasks.id)) AS total_count, COALESCE(SUM(sprint_member_tasks.points_earned),0) AS total_points_earned").
+		"COUNT(DISTINCT(tasks.id)) AS total_count, COALESCE(SUM(sprint_member_tasks.points_earned),0) AS total_points_earned, COALESCE(SUM(sprint_member_tasks.time_spent_minutes),0) AS actual_hours").
```
Comment (chahat-arora) [inline apps/retrospective/services/sprint.go]:
```suggestion
		`COUNT(DISTINCT(tasks.id)) AS total_count,
		 COALESCE(SUM(sprint_member_tasks.points_earned),0) AS total_points_earned, 
		 COALESCE(SUM(sprint_member_tasks.time_spent_minutes),0) AS actual_hours`).
```

**Review by chahat-arora — CHANGES_REQUESTED** [review]
Remove `dump.rdb` file.

File: apps/retrospective/services/sprint.go [hunk 2]
```diff
@@ -339,7 +340,9 @@ func (service SprintService) getSprintTaskTypeSummary(
 		QueryExpr()
 
 	taskQuery := taskList.Select(
-		"COUNT(DISTINCT(tasks.id)) AS total_count, COALESCE(SUM(sprint_member_tasks.points_earned),0) AS total_points_earned").
+		`COUNT(DISTINCT(tasks.id)) AS total_count, 
+	   COALESCE(SUM(sprint_member_tasks.points_earned),0) AS total_points_earned,
```
Comment (himanshu-garg-165) [inline apps/retrospective/services/sprint.go:343]:
```suggestion
		COALESCE(SUM(sprint_member_tasks.points_earned),0) AS total_points_earned,
```

**Review by himanshu-garg-165 — CHANGES_REQUESTED** [review]

File: apps/retrospective/services/sprint.go (hunk 1 above)
Comment (Ravi2714) [reply apps/retrospective/services/sprint.go]: done

File: apps/retrospective/services/sprint.go (hunk 2 above)
Comment (Ravi2714) [reply apps/retrospective/services/sprint.go:343]: done

File: apps/retrospective/serializers/sprint.go [hunk 3]
```diff
@@ -1,9 +1,10 @@
 package serializers
 
 import (
-	retroModels "github.com/iReflect/reflect-app/apps/retrospective/models"
 	"time"
 
+	retroModels "github.com/iReflect/reflect-app/apps/retrospective/models"
```
Comment (chahat-arora) [inline apps/retrospective/serializers/sprint.go:6]: sort imports properly.

File: apps/retrospective/services/sprint.go [hunk 4]
```diff
@@ -8,6 +8,8 @@ import (
 
 	"github.com/jinzhu/gorm"
 
+	"strings"
```
Comment (chahat-arora) [inline apps/retrospective/services/sprint.go]: sort imports properly.

**Review by chahat-arora — CHANGES_REQUESTED** [review]

File: apps/retrospective/serializers/sprint.go (hunk 3 above)
Comment (Ravi2714) [reply apps/retrospective/serializers/sprint.go:6]: done

File: apps/retrospective/services/sprint.go (hunk 4 above)
Comment (Ravi2714) [reply apps/retrospective/services/sprint.go]: done

**Review by chahat-arora — CHANGES_REQUESTED** [review]
Please, back merge develop.

**Review by himanshu-garg-165 — APPROVED** [review]

**Review by chahat-arora — APPROVED** [review]

**Review by himanshu-garg-165 — APPROVED** [review]

**Pull request merged**